boards/arduino/arduino_giga_r1/Kconfig.arduino_giga_r1

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ARDUINO_GIGA_R1_STM32H747XX_M7
+	bool
```

Comment:
@tejlmand, @nordicjm On dual core target, I had to leave (/add) the bool to board name definition, as the default generated (I guess it is generated in other cases) Kconfig BOARD symbol is `CONFIG_BOARD_STM32H747I_DISCO`.
Is that the expected way to proceed ?